app/components/standup_meeting/date_navigator_component.html.erb

```diff
@@ -1,45 +1,47 @@
-<div class="flex flex-row-reverse justify-between w-1/2 lg:w-1/4 mt-4 mx-auto md:mt-0 lg:mx-0 md:space-x-4" data-controller="link-builder" data-link-builder-param-value='date' data-link-builder-url-value="<%= request.path %>">
+<div class="flex flex-row-reverse mt-4 md:mt-0 lg:mx-0 md:space-x-4" data-controller="link-builder" data-link-builder-param-value='date' data-link-builder-url-value="<%= standup_meeting_group_standup_meetings_path(standup_meeting_group) %>">
   <% if next_standup_meeting? %>
     <%= link_to standup_meeting_group_standup_meetings_path(
       standup_meeting_group, 
       date: next_standup_meeting
-    ), class: 'link text-primary text-lg mt-2 ml-4 flex flex-col no-underline' do %>
+    ), class: 'hidden lg:flex link text-primary text-lg mt-2 ml-4 flex flex-col no-underline' do %>
       <i class="fa-solid fa-caret-right text-3xl"></i> 
-      <span class="md:hidden">
+      <span>
         Next
       </span>
     <% end %>
   <% else %>
-    <div class="text-lg mt-2 flex flex-col no-underline">
+    <div class="hidden lg:flex text-lg mt-2 flex-col no-underline">
       <i class="fa-solid fa-caret-right text-3xl text-base-300 cursor-not-allowed ml-4"></i> 
-      <span class="md:hidden">
+      <span>
         Next
       </span>
     </div>
   <% end %>
-  <a href="" class="btn hidden md:inline pt-4" data-link-builder-target='destination'>GO</a>
-  <%= text_field_tag :date_jumper, meeting_date, class: 'hidden md:inline border px-4 mx-2 w-48',data: {
-    controller: "flatpickr",
-    flatpickr_date_format: "Y-m-d",
-    flatpickr_min_date: standup_meeting_group.created_at,
-    action: "change->link-builder#updateDestinationParams",
-    link_builder_target: 'source'
-    } %>
+  <div class="flex justify-between mx-auto max-w-md">
+    <%= text_field_tag :date_jumper, meeting_date, class: 'inline border px-4 mx-2',data: {
+      controller: "flatpickr",
+      flatpickr_date_format: "Y-m-d",
```

Comment:
for now, we prob want to do m-d-Y cause that is what is most common on the us side.

Reply:
I had it in that format because it was easiest for `Date.parse` to pick up; in the updated PR, I've changed it to depend on a `strftime` version of m-d-Y.